docs/releasenotes/release-1.120.md

```diff
@@ -27,6 +27,7 @@ output fields from GCP APIs are in `status.observedState.*`
 * `MonitoringDashboard`
   * Added `style` fields to text widgets.
   * Added `sectionHeader` widgets.
+  * Added `dataSets.targetAxis` and `y2Axis` fields to `xyChart` widgets.
```

Comment:
super nit: alphabetical order to mitigate mergeconflict.

Reply:
Agreed.  I've tried to put the new widgets before the new fields in the widgets, but .. I agree